src/data/types/items.ts

```diff
@@ -13,22 +24,51 @@ export const ITEMS_SCHEMA = {
   electric_thermal_storage_and_slab: { type: 'string' },
   electric_vehicle_charger: { type: 'string' },
   electric_wiring: { type: 'string' },
+  energy_audit: { type: 'string' },
   evaporative_cooler: { type: 'string' },
+  floor_insulation: { type: 'string' },
   geothermal_heating_installation: { type: 'string' },
-  heat_pump_air_conditioner_heater: { type: 'string' },
+  heat_pump_air_conditioner_heater: { type: 'string' }, // TODO remove
   heat_pump_clothes_dryer: { type: 'string' },
   heat_pump_water_heater: { type: 'string' },
   new_electric_vehicle: { type: 'string' },
+  new_plugin_hybrid_vehicle: { type: 'string' },
   non_heat_pump_clothes_dryer: { type: 'string' },
   non_heat_pump_water_heater: { type: 'string' },
   other: { type: 'string' },
+  other_heat_pump: { type: 'string' },
```

Comment:
On second thought maybe this should be `other_hvac`? The only thing I can remember seeing in incentives that isn't covered by the other categories is window units, which I guess is technically a type of heat pump. I can't immediately think of a non-covered HVAC item that we would want to show that isn't some kind of heat pump.